charts/rancher-istio/1.5.920/questions.yaml

```
@@ -1,3 +1,4 @@
 labels:
   rancher.istio.v1.5.920: 1.5.9
 rancher_min_version: 2.6.0-alpha1
+rancher_max_version: 2.6.0-alpha1
```

```suggestion
rancher_max_version: 2.6.7
```

Let's not retroactively invalidate the chart. Instead, we can just make it invalid going forward.

---

@aiyengar2 Got it. I updated the `rancher_max_version` to 2.6.7. Can you please review it again?

---

@doflamingo721 2.6.8 was released yesterday 😅 , can you change it again?

---

@aiyengar2 I have updated the PR. Can you please review it again? TIA!

---

I think we should make it 2.6.99, since 2.6 release line will still contain the legacy feature and only starting in 2.7.0 we would remove it.  @aiyengar2 ?

---

Ah @prachidamle yes I think you're right. Based on the original issue https://github.com/rancher/rancher/issues/37327, we only want to hide the chart in 2.7, not in any 2.6 release.

Sorry @doflamingo721 can you change it to 2.6.99 as Prachi recommended?

---

@prachidamle @aiyengar2 I have updated the version to 2.6.99. Can you please review it again? Thanks!